openpype/hosts/blender/plugins/publish/validate_setdress_cam_instance.py

```diff
@@ -9,41 +9,29 @@
 from openpype.pipeline.publish import ValidateContentsOrder
 
 
-class ValidateSetdressCamInstance(pyblish.api.Validator):
-    """Validate that cam and review instances points on the right collection."""
+class ValidateReviewCamInstance(pyblish.api.Validator):
```

Comment:
I suggest we keep full names everywhere
```suggestion
class ValidateReviewCameraInstance(pyblish.api.Validator):
```